src/main/java/dev/sefiraat/cultivation/implementation/commands/CultivationCommands.java

```diff
@@ -205,10 +212,27 @@ private String getBlockId(Block block) {
         Material blockMaterial = block.getType();
         SlimefunItem slimefunItem = BlockStorage.check(block);
         return slimefunItem == null
-            ? "minecraft:" + blockMaterial.name()
-            : "slimefun:" + slimefunItem.getId();
+               ? "minecraft:" + blockMaterial.name()
+               : "slimefun:" + slimefunItem.getId();
     }
 
-
+    @Subcommand("removeEntities")
+    @CommandPermission("cultivation.admin.entities")
+    public void removeDisplayGroups(CommandSender sender, int radius) {
+        if (sender instanceof Player player) {
+            player.getWorld().getNearbyEntities(
+                player.getLocation(),
+                radius,
+                radius,
+                radius,
+                Interaction.class::isInstance
+            ).forEach(entity -> {
```

Comment:
i always got told of to no use .forEach()
not sure why

Reply:
Its slightly less performant but for a command like this, I'm cool with it, I also don't need any features forEach() locks you out of. :)